enchant.lic

```diff
@@ -77,20 +78,35 @@ class Enchant
     clean_brazier
     bput("get my #{@item}", 'You get a')
     2.times do
-      bput("put my #{@item} on #{@brazier}", 'You glance down', 'With a flick')
+      case bput("put my #{@item} on #{@brazier}", 'You glance down', 'With a flick', 'You must first clean')
+      when 'You must first clean'
+        echo "Something is still on the brazier.  Cannot continue"
+        exit
+      end
     end
     waitrt?
   end
 
   def imbue
-    get_item('imbue rod') unless left_hand.include?('rod')
-    case bput("wave rod at #{@item} on #{@brazier}", 'Roundtime', 'The streams collide, rending the space before you and disrupting the enchantment')
-    when 'The streams collide, rending the space before you and disrupting the enchantment'
-      imbue
+    imbue_data = @settings['waggle_sets']['imbue']['Imbue']
+    if imbue_data
+      crafting_prepare_spell(imbue_data,@settings)
+      castrt = checkcastrt - (imbue_data['cambrinth'].size * 3)
```

Comment:
❓ would it be easier to use `DRCA.crafting_cast_spell`?

Reply:
or maybe something like:
```ruby
imbue_spells = @settings['waggle_sets']['imbue']
cast_spells(imbue_spells, settings, settings.waggle_force_cambrinth)
```

Reply:
common-arcana doesn't have a method to cast at a particular target, which is what's required in this case

Reply:
true, that is a combat-trainer only thing

Reply:
Updated with the override to imbue_data['cast'].  Not sure if this is what you were thinking, but it worked in my test